naloge/poslovnez.tex

```diff
@@ -110,7 +110,56 @@
 \pgfslika[poslovnez-resitev2]
 \end{slika}
     
+\newpage
+\begin{small}
+\begin{algorithmic}
+\Function{MAKSIMALNIDOBIČEK}{$G = (V, E), s \in V, L : E \to \R$}
+    \State $d \gets$ seznam slovarjev z vrednostjo $- \infty$ za vsako vozlišče $v \in V$
+    \State $\pred \gets$ seznam slovarjev z vrednostjo $\Null$
```

Comment:
Tukaj naj bo jasno, kako indeksiraš sezname - piši torej `d[0, \dots, n]` in podobno za `\pred`, kjer lahko pravzaprav začneš z `1`.
```suggestion
    \State $d[0, \dots, n] \gets$ seznam slovarjev z vrednostjo $-\infty$ za vsako vozlišče $v \in V$
    \State $\pred[1, \dots, n] \gets$ seznam slovarjev z vrednostjo $\Null$
```